client/src/app/pages/dependencies/dependency-apps-table.tsx

```diff
@@ -0,0 +1,227 @@
+      {
+        key: "name",
+        title: "Application Name",
+        type: FilterType.search,
+        placeholderText:
+          t("actions.filterBy", {
+            what: "name", // TODO i18n
+          }) + "...",
+        getServerFilterValue: (value) => (value ? [`*${value[0]}*`] : []),
+      },
+      {
+        key: "businessService",
+        title: t("terms.businessService"),
+        placeholderText:
+          t("actions.filterBy", {
+            what: t("terms.businessService").toLowerCase(),
+          }) + "...",
+        type: FilterType.select,
+        selectOptions: businessServices
+          .map((businessService) => businessService.name)
+          .map((name) => ({ key: name, value: name })),
+      },
+      {
+        key: "tag.id",
+        title: t("terms.tags"),
+        type: FilterType.multiselect,
+        placeholderText:
+          t("actions.filterBy", {
+            what: t("terms.tagName").toLowerCase(),
+          }) + "...",
+        selectOptions: [...new Set(tags.map((tag) => tag.name))].map(
+          (tagName) => ({ key: tagName, value: tagName })
+        ),
+      },
+    ],
+    initialItemsPerPage: 10,
+  });
+
+  const {
+    result: { data: currentPageAppDependencies, total: totalItemCount },
+    isFetching,
+    fetchError,
+  } = useFetchAppDependencies(
+    getHubRequestParams({
+      ...tableControlState,
+      hubSortFieldKeys: {
+        name: "name",
+        version: "version",
+      },
+      implicitFilters: [
+        { field: "dep.name", operator: "=", value: dependency.name },
+        { field: "dep.version", operator: "=", value: dependency.version },
+        { field: "dep.sha", operator: "=", value: dependency.sha },
+      ],
+    })
+  );
+
+  const tableControls = useTableControlProps({
+    ...tableControlState,
+    idProperty: "name",
+    currentPageItems: currentPageAppDependencies,
+    totalItemCount,
+    isLoading: isFetching,
+    // TODO FIXME - we don't need selectionState but it's required by this hook?
+    selectionState: useSelectionState({
+      items: currentPageAppDependencies,
+      isEqual: (a, b) => a.name === b.name,
+    }),
+  });
+
+  const {
+    numRenderedColumns,
+    propHelpers: {
+      toolbarProps,
+      filterToolbarProps,
+      paginationToolbarItemProps,
+      paginationProps,
+      tableProps,
+      getThProps,
+      getTdProps,
+    },
+  } = tableControls;
+
+  return (
+    <>
+      <Toolbar {...toolbarProps} className={spacing.mtSm}>
+        <ToolbarContent>
+          <FilterToolbar {...filterToolbarProps} />
+          <ToolbarItem {...paginationToolbarItemProps}>
+            <SimplePagination
+              idPrefix="dependency-apps-table"
+              isTop
+              isCompact
+              paginationProps={paginationProps}
+            />
+          </ToolbarItem>
+        </ToolbarContent>
+      </Toolbar>
+      <TableComposable {...tableProps} aria-label="Affected files table">
```

Comment:
Ah, on closer inspection, this should be `aria-label="Affected apps table"` or something similar